quattor/types/metadata.pan

```diff
@@ -4,6 +4,10 @@ type structure_branch = {
     "author" ? string
     "name" : string
     "type" : string
+    @{a unique id, e.g. hash of last git commit}
+    "id" ? string
```

Comment:
which hosts? all hosts? 
i had a short discussion with @piojo about this, we were pondering the possibility of a new pan option to inject data from the commandline/ant task that would not be considered modified data wrt the decision making process that pan uses to determine which object templates to rebuild.
so only the object templates it would rebuild, would get the metadata. but it was over a beer, and certainly not while looking at the pan code :wink: